react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+class Home extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      images: [
+        { photoId: 0, url: 'images/001.png' },
+        { photoId: 1, url: 'images/004.png' },
+        { photoId: 2, url: 'images/007.png' },
+        { photoId: 3, url: 'images/025.png' },
+        { photoId: 4, url: 'images/039.png' }
+      ],
+      currentPhoto: 0
+    };
+    this.circleClick = this.circleClick.bind(this);
+    this.arrowClick = this.arrowClick.bind(this);
+  }
+
+  componentDidMount() {
+    this.carouselInterval = setInterval(() => {
+      if (this.state.currentPhoto === 4) {
```

Comment:
Instead of hard-coding a number here, try basing your logic off of the length of the array.